### PR TITLE
Fix simulator version on CI

### PR DESCRIPTION
**Pull request by osdnk**

### CODE REVIEW

**Review by brunobar79 — APPROVED** [review]
🎉

File: package.json [hunk 1]
```diff
@@ -343,7 +343,8 @@
       "simulator": {
         "type": "ios.simulator",
         "device": {
-          "type": "iPhone 12 Pro Max"
+          "type": "iPhone 11 Pro Max",
+          "os": "13.7"
```
Comment (skylarbarrera) [inline package.json:347]: why 13?

**Review by skylarbarrera — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (osdnk) [reply package.json:347]: because this one works and is available. This is not changing anything tbh, we're still building with the newest xcode.

Comment (skylarbarrera) [reply package.json:347]: yeah all good was jw

**Pull request merged**